### PR TITLE
fixed config for development/production. Added MongoStore

### PR DESCRIPTION
**Pull request by TheTrio**
#32 broke the local development setup since it mandated cookies to be secure. Since localhost is not secure, the cookies were not being set.

This fixes that. Have also added MongoStore for storing the user sessions in production

### CODE REVIEW

**Comment by TheTrio** [comment]
We need to set some environment variables before merging this though. Otherwise the automatic build will fail.

File: server/src/server/index.ts [hunk 1]
```diff
@@ -7,30 +7,46 @@ import passport from 'passport'
...
 app.set('views', path.join(__dirname, '../../src/views'))
 app.set('view engine', 'pug')
 
-const sessionMiddleware = session({
+const sessionConfig: session.SessionOptions = {
   saveUninitialized: true,
   resave: true,
   proxy: true,
-  secret: 'no_secret', //leaving this like this for now
+  secret: process.env.SECRET || 'superdupersecret', //leaving this like this for now
```
Comment (rushitote) [inline server/src/server/index.ts:28]:
```suggestion
  secret: process.env.SECRET || 'superdupersecret',
```
Now that this is fine...

**Review by rushitote — COMMENTED** [review]
lgtm

**Comment by TheTrio** [comment]
Have updated the environment variables. Merging this now.

**Pull request merged**